src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/PingHookProcessor.java

```diff
@@ -0,0 +1,15 @@
+package com.cloudbees.jenkins.plugins.bitbucket.hooks;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class PingHookProcessor extends HookProcessor {
+
+    private static final Logger LOGGER = Logger.getLogger(PingHookProcessor.class.getName());
+
+    @Override
+    public void process(HookEventType hookEvent, String payload, BitbucketType instanceType, String origin) {
+        LOGGER.log(Level.INFO, "Received webhook ping event from {0}", origin);
```

Comment:
Looking forward to this 😅

Reply:
Hi, 
When could i expect that this PR will be merged and new version of the plugin will be released?
I really need this since now my integration bitbucket<->jenkins does not work.

Reply:
Not too soon, I guess. As stated in the title, it's a work in progress. There are still some features missing, and some comments to address. That said, if you wanna try it out, I'd encourage to install a custom build of the plugin into your Jenkins installation and provide feedback if it worked, or what did not work 😊